Makefile

```diff
@@ -6,6 +6,15 @@ ROOTDIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 # CircleCI yarn cache directory may also need to be updated in sync with this
 YARN_CACHE_DIR=~/.yarncache
 
+ISOLATED_VM_VERSION_COMMAND="require('./node_modules/isolated-vm/package.json').version"
```

Comment:
Little unclear on how the versions will change. This seems to be checking the version that's installed in the main project, which can get updated by dependabot. The separate package.json in this file is at a frozen version. Isn't the frozen version all that matters? Or we're ok letting dependabot update the version for us, and it triggers us building a new version for lambda, which will get used the next time we bump the SDK in `coda`?

Reply:
oh package.json doesn't specify a fixed version. it uses something like ^4.3.2 and allows a patch version difference from yarn install. the idea is basically to use the version from yarn.lock which is more precise. 

instead of parsing yarn.lock, `require('./node_modules/isolated-vm/package.json').version` gives us the same version number.